packages/cli/src/util/build/write-build-result.ts

```diff
@@ -235,7 +235,20 @@ async function writeStaticFile(
   const dest = join(outputDir, 'static', fsPath);
   await fs.mkdirp(dirname(dest));
 
-  // TODO: handle (or skip) symlinks?
+  // if the source is a symlink, try to create it instead of copying the file
+  // however, if the symlink creation fails (e.g. Windows), fallback to copying
+  if (
+    file.type === 'FileFsRef' &&
```

Comment:
This is correct, but only handles the `FileFsRef` case. To be complete, we should handle the other `File` type implementations.

Thinking about it a bit more now, I realize that we already have the necessary logic [in this function](https://github.com/vercel/vercel/blob/a82f1172179b4906b35b005e7a45b27a760fd82f/packages/build-utils/src/fs/download.ts#L45-L57). So perhaps let's just export that function from build-utils and we can use it here.